### PR TITLE
Update removing-a-member-from-your-organization.md

### PR DESCRIPTION
**Pull request by datanerd**
### Why:

Closes [issue link]

### What's being changed:

The page doesn't mention what happens to forks of internal repositories. Similarly to forks of private repositories, forks of internal repositories will be removed from user's account if they are removed from the organisation. However, if the user is a member of another orgnisation under the same enterprise account, they will still have the access until they are removed from ALL organisations under that enterprise account.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

/cc @affrae

### CODE REVIEW

**Comment by andrekolodochka** [comment]
I will need help with converting absolute `https://docs.github.com/en/github/setting-up-and-managing-your-enterprise/managing-your-enterprise-account/about-enterprise-accounts` link

File: content/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Only organization owners can remove members from an organization.
...
 - Removed members will lose access to private forks of your organization's private repositories, but they may still have local copies. However, they cannot sync local copies with your organization's repositories. Their private forks can be restored if the user is [reinstated as an organization member](/articles/reinstating-a-former-member-of-your-organization) within three months of being removed from the organization. Ultimately, you are responsible for ensuring that people who have lost access to a repository delete any confidential information or intellectual property.
+- For customers with [Enterprise Account](https://docs.github.com/en/github/setting-up-and-managing-your-enterprise/managing-your-enterprise-account/about-enterprise-accounts), removed members will lose access to private forks of your organization's internal repositories, but only when the removed member is not a member of another organization under that Enterprise Account.
```
Comment (affrae) [inline content/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization.md:26]:
```suggestion
- In addition, for customers with [Enterprise Accounts](https://docs.github.com/en/github/setting-up-and-managing-your-enterprise/managing-your-enterprise-account/about-enterprise-accounts), removed members will lose access to private forks of your organization's internal repositories in this manner when and only when the removed member is not a member of another organization under that Enterprise Account.
```

File: content/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization.md [hunk 2]
```diff
@@ -33,6 +34,7 @@ Only organization owners can remove members from an organization.
 
 **Warning:** When you remove members from an organization:
  - Removed members will lose access to private forks of your organization's private repositories, but may still have local copies. However, they cannot sync local copies with your organization's repositories. Their private forks can be restored if the user is [reinstated as an organization member](/articles/reinstating-a-former-member-of-your-organization) within three months of being removed from the organization. Ultimately, you are responsible for ensuring that people who have lost access to a repository delete any confidential information or intellectual property.
+ - For customers with [Enterprise Account](https://docs.github.com/en/github/setting-up-and-managing-your-enterprise/managing-your-enterprise-account/about-enterprise-accounts), removed members will lose access to private forks of your organization's internal repositories, but only when the removed member is not a member of another organization under that Enterprise Account.
```
Comment (affrae) [inline content/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization.md:37]:
```suggestion
 - In addition, for customers with [Enterprise Accounts](https://docs.github.com/en/github/setting-up-and-managing-your-enterprise/managing-your-enterprise-account/about-enterprise-accounts), removed members will lose access to private forks of your organization's internal repositories in this manner when and only when the removed member is not a member of another organization under that Enterprise Account.
```

**Review by affrae — COMMENTED** [review]
Some minor suggestions but not using this to block approval.

**Comment by ramyaparimi** [comment]
@andrekolodochka 
Thanks so much for opening a PR! Not sure how I missed this, my notifications are probably a little wonky. I'll get this triaged for review :zap:

**Comment by sksahalam786** [comment]
hello

File: content/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization.md [hunk 3]
```diff
@@ -23,6 +23,7 @@ Only organization owners can remove members from an organization.
...
 - Removed members will lose access to private forks of your organization's private repositories, but they may still have local copies. However, they cannot sync local copies with your organization's repositories. Their private forks can be restored if the user is [reinstated as an organization member](/articles/reinstating-a-former-member-of-your-organization) within three months of being removed from the organization. Ultimately, you are responsible for ensuring that people who have lost access to a repository delete any confidential information or intellectual property.
+- In addition, for customers with [Enterprise Accounts](https://docs.github.com/en/github/setting-up-and-managing-your-enterprise/managing-your-enterprise-account/about-enterprise-accounts), removed members will lose access to private forks of your organization's internal repositories in this manner when and only when the removed member is not a member of another organization under that Enterprise Account.
```
Comment (lecoursen) [inline content/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization.md:26]: For GHES and GHAE, all organizations are owned by an enterprise account, so I'm versioning the mentions of enterprise accounts for dotcom only.
```suggestion
- {% ifversion fpt %}If your organization is owned by an enterprise account, removed{% else %}Removed{% endif %} members will also lose access to private forks of your organization's internal repositories, if the removed member is not a member of any other organization{% ifversion fpt %} owned by the same enterprise account. For more information, see "[About enterprise accounts](/github/setting-up-and-managing-your-enterprise/managing-your-enterprise-account/about-enterprise-accounts)."{% else %}.{% endif %}
```

File: content/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization.md [hunk 4]
```diff
@@ -33,6 +34,7 @@ Only organization owners can remove members from an organization.
 
 **Warning:** When you remove members from an organization:
  - Removed members will lose access to private forks of your organization's private repositories, but may still have local copies. However, they cannot sync local copies with your organization's repositories. Their private forks can be restored if the user is [reinstated as an organization member](/articles/reinstating-a-former-member-of-your-organization) within three months of being removed from the organization. Ultimately, you are responsible for ensuring that people who have lost access to a repository delete any confidential information or intellectual property.
+ - In addition, for customers with [Enterprise Accounts](https://docs.github.com/en/github/setting-up-and-managing-your-enterprise/managing-your-enterprise-account/about-enterprise-accounts), removed members will lose access to private forks of your organization's internal repositories in this manner when and only when the removed member is not a member of another organization under that Enterprise Account.
```
Comment (lecoursen) [inline content/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization.md:37]:
```suggestion
- {% ifversion fpt %}If your organization is owned by an enterprise account, removed{% else %}Removed{% endif %} members will also lose access to private forks of your organization's internal repositories, if the removed member is not a member of any other organization{% ifversion fpt %} owned by the same enterprise account. For more information, see "[About enterprise accounts](/github/setting-up-and-managing-your-enterprise/managing-your-enterprise-account/about-enterprise-accounts)."{% else %}.{% endif %}
```

**Review by lecoursen — COMMENTED** [review]
Thanks @andrekolodochka, this is really helpful to clarify! I'm applying some small changes to add versioning around the content about enterprise accounts, and then I'll merge. ⚡

File: content/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization.md [hunk 5]
```diff
@@ -23,6 +23,7 @@ Only organization owners can remove members from an organization.
...
 - Removed members will lose access to private forks of your organization's private repositories, but they may still have local copies. However, they cannot sync local copies with your organization's repositories. Their private forks can be restored if the user is [reinstated as an organization member](/articles/reinstating-a-former-member-of-your-organization) within three months of being removed from the organization. Ultimately, you are responsible for ensuring that people who have lost access to a repository delete any confidential information or intellectual property.
+- {% ifversion fpt %}If your organization is owned by an enterprise account, removed{% else %}Removed{% endif %} members will also lose access to private forks of your organization's internal repositories, if the removed member is not a member of any other organization{% ifversion fpt %} owned by the same enterprise account. For more information, see "[About enterprise accounts](/github/setting-up-and-managing-your-enterprise/managing-your-enterprise-account/about-enterprise-accounts)."{% else %}.{% endif %}
```
Comment (lecoursen) [inline content/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization.md:26]:
```suggestion
-  If your organization is owned by an enterprise account, removed members will also lose access to private forks of your organization's internal repositories, if the removed member is not a member of any other organization owned by the same enterprise account. For more information, see "[About enterprise accounts](/github/setting-up-and-managing-your-enterprise/managing-your-enterprise-account/about-enterprise-accounts)."
```

File: content/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization.md [hunk 6]
```diff
@@ -33,6 +34,7 @@ Only organization owners can remove members from an organization.
 
 **Warning:** When you remove members from an organization:
  - Removed members will lose access to private forks of your organization's private repositories, but may still have local copies. However, they cannot sync local copies with your organization's repositories. Their private forks can be restored if the user is [reinstated as an organization member](/articles/reinstating-a-former-member-of-your-organization) within three months of being removed from the organization. Ultimately, you are responsible for ensuring that people who have lost access to a repository delete any confidential information or intellectual property.
+- {% ifversion fpt %}If your organization is owned by an enterprise account, removed{% else %}Removed{% endif %} members will also lose access to private forks of your organization's internal repositories, if the removed member is not a member of any other organization{% ifversion fpt %} owned by the same enterprise account. For more information, see "[About enterprise accounts](/github/setting-up-and-managing-your-enterprise/managing-your-enterprise-account/about-enterprise-accounts)."{% else %}.{% endif %}
```
Comment (lecoursen) [inline content/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization.md:37]:
```suggestion
- Removed members will also lose access to private forks of your organization's internal repositories, if the removed member is not a member of any other organization in your enterprise.
```

**Review by lecoursen — APPROVED** [review]

**Pull request merged**